src/layman/authz/role_service.py

```diff
@@ -38,3 +38,47 @@ def get_all_roles():
     """
     roles = db_util.run_query(query, ('ADMIN', 'GROUP_ADMIN', settings.LAYMAN_GS_ROLE, ROLE_NAME_PATTERN), uri_str=settings.LAYMAN_ROLE_SERVICE_URI)
     return [role[0] for role in roles] + [settings.RIGHTS_EVERYONE_ROLE]
+
+
+def validate_role_table():
+    expected_roles = ['ADMIN', 'GROUP_ADMIN', settings.LAYMAN_GS_ROLE]
+    query = f"""
+    select unnest(%s)
+    EXCEPT
+    select name
+    from {settings.LAYMAN_ROLE_SERVICE_SCHEMA}.roles
+    """
+    roles = db_util.run_query(query, (expected_roles,), uri_str=settings.LAYMAN_ROLE_SERVICE_URI)
+    if roles:
+        raise Exception(f"Missing roles in JDBC Role service: {[role[0] for role in roles]}")
+
+    query = f"""
+    select name
+    from {settings.LAYMAN_ROLE_SERVICE_SCHEMA}.roles
+    where name !~ %s
+    """
+    roles = db_util.run_query(query, (ROLE_NAME_PATTERN,), uri_str=settings.LAYMAN_ROLE_SERVICE_URI)
+    if roles:
+        raise Exception(f"Roles not matching pattern '{ROLE_NAME_PATTERN}' in JDBC Role service: {[role[0] for role in roles]}")
+
+    query = f"""
+    select name
+    from {settings.LAYMAN_ROLE_SERVICE_SCHEMA}.roles
+    where name = %s
+    """
+    roles = db_util.run_query(query, (settings.RIGHTS_EVERYONE_ROLE,), uri_str=settings.LAYMAN_ROLE_SERVICE_URI)
+    if roles:
+        raise Exception(f"Role '{settings.RIGHTS_EVERYONE_ROLE}' should not be in JDBC Role service.")
```

Comment:
Ensure that also roles ROLE_ADMINISTRATOR, ROLE_GROUP_ADMIN, ROLE_AUTHENTICATED, ROLE_ANONYMOUS are not in the table (according to specification).